### PR TITLE
Run clang-format

### PR DESCRIPTION
**Pull request by BillThePlatypus**
Running clang-format so that we can have a consistent style.

### CODE REVIEW

File: examples/i2c_airspeed/main.cpp [hunk 1]
```diff
@@ -86,15 +86,14 @@ int main() {
...
-    
+
     if (millis() > last_print_ms + 50)
     {
-      last_print_ms = millis();        
-      printf("%d.%dPa, %d.%dC\n",
-               (int32_t)(diff_press), (int32_t)(diff_press*1000)%1000,
-               (int32_t)(temp), (int32_t)(temp*1000)%1000);
+      last_print_ms = millis();
+      printf("%d.%dPa, %d.%dC\n", (int32_t)(diff_press), (int32_t)(diff_press * 1000) % 1000, (int32_t)(temp),
```
Comment (superjax) [inline examples/i2c_airspeed/main.cpp]: could you wrap this in `// clang-format off` /`on` blocks?

File: examples/i2c_baro/main.cpp [hunk 2]
```diff
@@ -68,19 +68,16 @@ int main() {
...
     {
       warn.off();
       info.toggle();
       baro.read(&pressure, &temperature);
-      printf("%d Pa, %d.%d K\n",
-             (int32_t)(pressure),
-             (int32_t)(temperature),
-             (int32_t)(temperature*100)%100);
-      
+      printf("%d Pa, %d.%d K\n", (int32_t)(pressure), (int32_t)(temperature), (int32_t)(temperature * 100) % 100);
```
Comment (superjax) [inline examples/i2c_baro/main.cpp]: same, revert this change

File: examples/i2c_mag/main.cpp [hunk 3]
```diff
@@ -65,27 +65,23 @@ int main() {
...
     if (mag.present())
     {
       warn.off();
       info.toggle();
       mag.read(mag_data);
-      printf("%d, %d, %d\n",
-             (int32_t)(mag_data[0]),
-             (int32_t)(mag_data[1]),
-             (int32_t)(mag_data[2]));
+      printf("%d, %d, %d\n", (int32_t)(mag_data[0]), (int32_t)(mag_data[1]), (int32_t)(mag_data[2]));
```
Comment (superjax) [inline examples/i2c_mag/main.cpp]: fence and undo

File: examples/spi/main.cpp [hunk 4]
```diff
@@ -81,15 +80,9 @@ int main() {
...
-      printf("%d\t%d\t%d\t%d\t%d\t%d\t%d\t%d\n",
-             (int32_t) (acc[0]*1000.0),
-             (int32_t) (acc[1]*1000.0),
-             (int32_t) (acc[2]*1000.0),
-             (int32_t) (gyro[0]*1000.0),
-             (int32_t) (gyro[1]*1000.0),
-             (int32_t) (gyro[2]*1000.0),
-             (int32_t) (temp*1000.0),
-             time_us);
+      printf("%d\t%d\t%d\t%d\t%d\t%d\t%d\t%d\n", (int32_t)(acc[0] * 1000.0), (int32_t)(acc[1] * 1000.0),
```
Comment (superjax) [inline examples/spi/main.cpp]: fence and undo

**Review by superjax — APPROVED** [review]
Let's undo some of the printing formatting changes, but otherwise LGTM

**Comment by plusk01** [comment]
i'd suggest ignoring all the `lib` stuff to make future comparisons easier
